RELEASE_NOTES.md

```diff
@@ -1,4 +1,6 @@
-#### 1.3.1.0 -
+#### 1.3.2 -
```

Comment:
The "3.1" was intended to refer to the language version supported (1.3.1.x v. 1.4.0.x) . Not exactly SemVer I know. Can you use 1.3.1.1 or else suggest a different version numbering stream for the 3.1 and 4.0 versions? thx